content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -1,110 +1,129 @@
+section, and select :guilabel:`Received quantities`. Then, :guilabel:`Save` changes.
+
+In the :guilabel:`Purchase` app, create a new :abbr:`RFQ (Request for Quotation)`. Fill out the
+information on the quotation, add products to the invoice lines, and click
+:guilabel:`Confirm Order`. Then, click on the :guilabel:`Receipt smart button`. Set the quantities
+in the :guilabel:`Done` column to match the quantities in the :guilabel:`Demand` column, and
+:guilabel:`Validate` the changes. Then, in the purchase order, click :guilabel:`Create Bill` and
+:guilabel:`Confirm`. Since the policy is set to *received quantities*, the draft bill can be
+confirmed *only* when at least some of the quantities are received.
+
+3-way matching
+==============
+
+Activating :guilabel:`3-way matching` ensures that vendor bills are only paid once some or all of
+the products included in the purchase order have actually been received. To activate it, go to
+:menuselection:`Purchase --> Configuration --> Settings`, and scroll down to the
+:guilabel:`Invoicing` section. Then, click
+:guilabel:`3-way matching: purchases, receptions, and bills`.
 
-+--------------------+--------------------------------------------------------------------------+
-| **Billing status** |                              **Conditions**                              |
-|                    +------------------------------------------------+-------------------------+
-|                    |            *On received quantities*            | *On ordered quantities* |
-+--------------------+------------------------------------------------+-------------------------+
-| *Nothing to Bill*  |       PO confirmed; no products received       |    *(Not applicable)*   |
-+--------------------+------------------------------------------------+-------------------------+
-| *Waiting Bills*    |  All/some products received; bill not created  |       PO confirmed      |
-+--------------------+------------------------------------------------+-------------------------+
-| *Fully Billed*     | All/some products received; draft bill created |    Draft bill created   |
-+--------------------+------------------------------------------------+-------------------------+
+.. note::
+   :guilabel:`3-way matching` is *only* intended to work when the bill control policy is set to
+   :guilabel:`Received quantities`.
 
-Determine when to pay with 3-way matching
-=========================================
+Pay vendor bills with 3-way matching
+------------------------------------
 
-First, go to :menuselection:`Configuration --> Settings` and activate *3-way matching*.
+When :guilabel:`3-way matching` is activated, vendor bills will display the
+:guilabel:`Should Be Paid` field under the :guilabel:`Other Info` tab. When a new vendor bill is
+created, the field will be set to :guilabel:`Yes`, since a bill can't be created until at least
+some of the products included in a purchase order have been received.
```

Comment:
```suggestion
When :guilabel:`3-way matching` is activated, vendor bills will display the :guilabel:`Should Be
Paid` field under the :guilabel:`Other Info` tab. When a new vendor bill is created, the field will
be set to :guilabel:`Yes`, since a bill can't be created until at least some of the products
included in a purchase order have been received.
```